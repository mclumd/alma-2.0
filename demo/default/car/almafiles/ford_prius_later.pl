obs(quote(rel(car, ford))).
obs(quote(rel(car, prius))).

obs(quote(not(rel(fueled, ford)))).
obs(quote(rel(charged, prius))).
obs(quote(not(rel(fueled, prius)))).


fif(now(3), obs(quote(rel(gas_tank_car, ford)))).

fif(now(3), obs(quote(rel(electric_car, prius)))).
fif(now(3), obs(quote(rel(gas_tank_car, prius)))).


fif(now(14), obs(quote(rel(car, ford2)))).
fif(now(14), obs(quote(rel(car, prius2)))).

fif(now(14), obs(quote(not(rel(fueled, ford2))))).
fif(now(14), obs(quote(rel(charged, prius2)))).
fif(now(14), obs(quote(not(rel(fueled, prius2))))).


fif(now(17), obs(quote(rel(gas_tank_car, ford2)))).

fif(now(17), obs(quote(rel(electric_car, prius2)))).
fif(now(17), obs(quote(rel(gas_tank_car, prius2)))).
