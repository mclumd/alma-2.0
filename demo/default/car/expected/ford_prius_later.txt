rel(car, ford)
rel(gas_tank_car, ford)
~rel(powered, ford)
~rel(fueled, ford)
abnormal(ford, car, "~rel(powered, ford)")
rel(car, prius)
rel(gas_tank_car, prius)
rel(electric_car, prius)
rel(charged, prius)
~rel(fueled, prius)
rel(powered, prius)
abnormal(prius, gas_tank_car, "rel(powered, prius)")
rel(car, ford2)
rel(gas_tank_car, ford2)
~rel(powered, ford2)
~rel(fueled, ford2)
abnormal(ford2, car, "~rel(powered, ford2)")
rel(car, prius2)
rel(gas_tank_car, prius2)
rel(electric_car, prius2)
rel(charged, prius2)
~rel(fueled, prius2)
rel(powered, prius2)
abnormal(prius2, gas_tank_car, "rel(powered, prius2)")
